src/include/platform/DiagnosticDataProvider.h

```diff
@@ -110,6 +110,7 @@ class DiagnosticDataProvider
     virtual CHIP_ERROR GetActiveHardwareFaults(GeneralFaults<kMaxHardwareFaults> & hardwareFaults);
     virtual CHIP_ERROR GetActiveRadioFaults(GeneralFaults<kMaxRadioFaults> & radioFaults);
     virtual CHIP_ERROR GetActiveNetworkFaults(GeneralFaults<kMaxNetworkFaults> & networkFaults);
+    virtual CHIP_ERROR GetAverageWearCount(uint32_t & averageWearCount);
```

Comment:
This needs API documentation.   Yes, I know nothing else in this interface has it....

Reply:
Done.